### PR TITLE
docs(react): update doc for ErrorBoundary

### PR DESCRIPTION
**Pull request by manudeli**
related with #179 

# Overview

I updated doc for ErrorBoundary

## PR Checklist

- [x] I have written documents and tests, if needed.

### CODE REVIEW

File: packages/react/src/ErrorBoundary.en.mdx [hunk 1]
```diff
@@ -3,16 +3,15 @@ sidebar_position: 2
 title: <ErrorBoundary/>
 ---
 
-This component provide a simple and reusable wrapper that you can use to wrap around your components. Any rendering errors in your components hierarchy can then be gracefully handled.
+Any rendering errors in children of this component can be handled.
```
Comment (okinawaa) [inline packages/react/src/ErrorBoundary.en.mdx:6]: Due to the `useErrorboundary`, errors other than rendering can be handled in the `Errorboundary`, is it okay not to specify that part?

File: packages/react/src/ErrorBoundary.en.mdx [hunk 2]
```diff
@@ -3,16 +3,15 @@ sidebar_position: 2
...
 
-![Example banner](/gif/errorboundary-example.gif)
-
-## fallback
+### props.fallback
 
-If there is any thrown error in children, ErrorBoundary will catch it and then fallback will be rendered.
+If there is any thrown error in children of `<ErrorBoundary/>`, Error will be catched and then fallback will be rendered.
```
Comment (okinawaa) [inline packages/react/src/ErrorBoundary.en.mdx:10]:
```suggestion
If there is any thrown error in children of `<ErrorBoundary/>`, Error will be caught and then fallback will be rendered.
```

File: packages/react/src/ErrorBoundary.en.mdx [hunk 3]
```diff
@@ -81,9 +120,18 @@ const Example = () => (
 )
 ```
 
-We can use useErrorBoundary.setError in ErrorBoundaryFallback to make ErrorBoundary aware of the Error without throw.
+We can use useErrorBoundary.setError in ErrorBoundaryFallback to make `<ErrorBoundary/>` aware of the Error without throw.
```
Comment (okinawaa) [inline packages/react/src/ErrorBoundary.en.mdx:123]: actually I usually use `setError` in children, not in ErrorBoundaryFallback😄

I think I'll use useErrorboundary.reset a lot in fallback component

File: packages/react/src/ErrorBoundary.en.mdx (hunk 2 above)
Comment (manudeli) [reply packages/react/src/ErrorBoundary.en.mdx:10]: You caught wrong grammer! Thanks

File: packages/react/src/ErrorBoundary.en.mdx (hunk 3 above)
Comment (manudeli) [reply packages/react/src/ErrorBoundary.en.mdx:123]: Oh! you're right, It's my mistake

File: packages/react/src/ErrorBoundary.en.mdx (hunk 1 above)
Comment (manudeli) [reply packages/react/src/ErrorBoundary.en.mdx:6]: Wow... so good catch!👍

**Review by okinawaa — APPROVED** [review]

**Pull request merged**